__tests__/app.test.js

```diff
@@ -79,6 +79,41 @@ describe("/api", () => {
   });
 });
 
+describe("/api/reviews/:review_id", () => {
+  test("GET - status: 200 - returns an object of review id has corresponding properties", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviewId[0].hasOwnProperty("review_id")).toBe(true);
```

Comment:
Ideally we should assert that the correct review is sent back so we can check for specifics rather than the general assertions you have here. Please update this test to do so 😄